Meta/CMake/common_options.cmake

```diff
@@ -26,7 +26,7 @@ serenity_option(ENABLE_ACCELERATED_GRAPHICS ON CACHE BOOL "Enable use of acceler
 
 serenity_option(HACKSTUDIO_BUILD OFF CACHE BOOL "Automatically enabled when building from HackStudio")
 
-serenity_option(ENABLE_JAKT OFF CACHE BOOL "Enable building jakt files")
+serenity_option(ENABLE_JAKT ON CACHE BOOL "Enable building jakt files")
```

Comment:
I think this currently does all of nothing lol

Reply:
reset to default off? until we have something fun to show.